dao/src/main/kotlin/repositories/ortrun/DaoOrtRunRepository.kt

```diff
@@ -176,6 +177,12 @@ class DaoOrtRunRepository(private val db: Database) : OrtRunRepository {
     override fun delete(id: Long): Int = db.blockingQuery {
         OrtRunsTable.deleteWhere { OrtRunsTable.id eq id }
     }
+
+    override fun deleteAll(ids: List<Long>): Int = db.blockingQuery {
+        if (ids.isEmpty()) return@blockingQuery 0
+
+        OrtRunsTable.deleteWhere { OrtRunsTable.id inList ids }
+    }
```

Comment:
How about making a function to delete ORT runs by repository id instead? I believe that would make the implementation simpler on the caller side, because you don't have to fetch the list of runs first. Also, I don't know what the limitation for a list in a query is, but this implementation might fail if `ids` becomes too large.